src/codemagic/tools/firebase_app_distribution/action_groups/releases_action_group.py

```diff
@@ -43,6 +44,10 @@ def list_releases(
         except GoogleError as e:
             raise FirebaseAppDistributionError(str(e))
 
-        printer = ResourcePrinter(json_output, self.echo)
-        printer.print_resources(releases, should_print)
+        if not releases:
+            self.logger.info(Colors.YELLOW(f"No releases available for {app_identifier.app_id}"))
+        else:
+            printer = ResourcePrinter(json_output, self.echo)
+            printer.print_resources(releases, should_print)
+
```

Comment:
```suggestion
        if not releases:
            self.logger.info(Colors.YELLOW(f"No releases available for {app_identifier.app_id}"))
            return []

        printer = ResourcePrinter(json_output, self.echo)
        printer.print_resources(releases, should_print)

```

Reply:
In expected scenarios this equivalent to current implementations. As with the [comment above](https://github.com/codemagic-ci-cd/cli-tools/pull/373/files#r1390794781), if the `releases` indeed comes out as `None` (or any other falsy equivalent), then it is better if that unexpected value somehow surfaces.